src/VerifierAlgorithm.ts

```diff
@@ -1,12 +1,10 @@
-import { ec as EC, SignatureInput } from 'elliptic'
-import { sha256, toEthereumAddress } from './Digest'
-import { verify } from '@stablelib/ed25519'
 import type { VerificationMethod } from 'did-resolver'
-import { bases } from 'multiformats/basics'
-import { hexToBytes, base58ToBytes, base64ToBytes, bytesToHex, EcdsaSignature, stringToBytes } from './util'
-import { verifyBlockchainAccountId } from './blockchains'
+import { base64ToBytes, bytesToHex, EcdsaSignature } from './util'
 
-const secp256k1 = new EC('secp256k1')
+import * as Ed25519VerifierAlg from './VerifierAlg/Ed25519VerifierAlg'
```

Comment:
flagging this for simplification of imports:
```suggestion
import { Ed25519VerifierAlg } from './VerifierAlg/Ed25519VerifierAlg'
```

Reply:
just to be clearer, all these imports should be simplified this way, not just this line